.vscode/launch.json

```diff
@@ -7,6 +7,7 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
+                "${workspaceRoot}/test/integrationTests/testAssets/slnWithCsproj", 
```

Comment:
I'm still not a super fan of this particular change because it'll make it more of a pain to debug customer repros. I've often had a customer repro project and relied on the fact that subsequent F5 invocations (after the first one) would load the previously loaded workspace. This would require me to change launch.json to get that behavior.

Reply:
Ok, I'll revert this to unblock your workflow.

Reply:
Thanks! I think it looks good to merge otherwise.